src/components/card/NFTCard.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./NFT.styles.css";
```

Comment:
You have mui, css, and inline -css all in one component. Try to work with mui and use styled component if you need to override default design